src/ex5/server/db/models/items.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class Items extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  Items.init(
+    {
+      ItemName: DataTypes.STRING,
+      status: DataTypes.BOOLEAN,
+    },
+    {
+      sequelize,
+      modelName: "Items",
```

Comment:
This is where I meant that you can add the underscored:true
```suggestion
      modelName: "Items",
      underscored: true,
```